routes/apps.js

```diff
@@ -1,4 +1,45 @@
 const express = require('express');
 const router = express.Router();
 
+//Requires to have the apps.json file on server
+//Also we create the array
+const appsData = require('../data/apps.json');
+
+//Using router object and express function Get
+// we send data to json 
+router.get('/', function(req, res, next) {
+    res.json(appsData);
+});
+
+router.get('/:id', function(req, res, next) {
+    var parsed = parseInt(req.params.id);
+    var found = new Object();
+
+    // We parse the ID number from the request and 
+    // looking for a matching number in the array of 
+    // apps we created using while loop. If we have a match,
+    // we return it as an Object
+
+    var idx = 0;
+    while (idx < appsData.length) {
+        if (appsData[idx].id === parsed) {
+            found = appsData[idx];  
+            break;     
+        }
+        idx++;
+    }
+    
```

Comment:
Good work! It's not incorrect, but there's a simpler, more JavaScripty way. 

` res.status(200).send(appsData.find(element => element.id === parseInt(req.params.id)) || {});`

Again, not incorrect, just an FYI. Good work!